source/custom/xnetsocket_converters.h

```diff
@@ -327,6 +346,10 @@ struct SockOptDataInputConverter {
         break;
       case SockOptData::DataCase::kDataLinger:
         return sizeof(data_linger);
+      case SockOptData::DataCase::kDataIpmreq:
```

Comment:
nit: I think we would want this to be kDataIpMreq

Reply:
Updated names so that we get this name for the Datacase